test/seneca.test.js

```diff
@@ -378,14 +381,17 @@ describe(__filename, function(){
         assert.ok( o.foo.entry )
 
         si.add({op:'foo'},bar)
+				// console.log('=============')
```

Comment:
more

Reply:
there were actually a few more - must have been lurking in there for years ago!
cleaned them all up